xarray/tests/test_duck_array_ops.py

```diff
@@ -571,3 +572,46 @@ def test_docs():
             indicated dimension(s) removed.
         """)
     assert actual == expected
+
+
+def test_datetime_to_numeric_datetime64():
```

Comment:
It might make sense to modify these tests now that `duck_array_ops.datetime_to_numeric` is only meant to take NumPy or dask arrays as input (these tests are using DataArrays).